### PR TITLE
Add hydpy group link section

### PR DESCRIPTION
**Pull request by vamc-k**
**Summary:**
 - Added organizers section with title, description and logo.

**Issue Link:**

 - #13 

**Screenshots:**

image

image

image

image

**PR Checks:**

- [x] Code is properly formatted (executed `make pre-push` command).
- [x] No ESLint warnings (verified with `make pre-push` command).
- [x] UI Changes are responsive (checked in the responsive view).
- [x] Performance is maintained (Lighthouse report reviewed).
- [x] User experience (UX) considerations are met.
- [ ] Accessibility is supported (validated with accessibility tools).

### CODE REVIEW

**Comment by vamc-k** [comment]
@RajatRajdeep  please review this PR and let me know if any changes are required.

File: src/components/MeetOrganizers.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+      <div className="flex flex-col justify-center items-center gap-3">
+        <Image
+          src={ABOUT_HYDPY.hydPyLogoUrl}
+          alt="Event Design"
+          className="rounded-lg"
+          width={400}
+          height={400}
+        />
+        <p className="text-lg md:text-xl mb-8">
```
Comment (RajatRajdeep) [inline src/components/MeetOrganizers.jsx]: Could you use [Link](https://nextjs.org/docs/pages/api-reference/components/link) here?
This is the recommended in next.js for redirection.

File: data/details.js [hunk 2]
```diff
@@ -23,6 +23,15 @@ export const HERO_BANNER = {
   cfpUrl: 'https://www.papercall.io/pyconfhyd2025',
 };
 
+export const ABOUT_HYDPY = {
+  title: 'Meet our Organizers',
```
Comment (RajatRajdeep) [inline data/details.js]: `Meet the Organizers: HydPy` sounds more fitting to me. Can we use this instead?

File: src/components/MeetOrganizers.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { ABOUT_HYDPY } from '@/details';
+import Image from 'next/image';
+
+export default function MeetOrganizers() {
+  return (
+    <div className="flex flex-col lg:flex-row justify-center items-center p-6 pt-12 pb-12 border-t-2 border-gray-300">
+      <div className="max-w-3xl lg:mr-20">
+        <h1 className="text-4xl font-bold text-[#001ccf] dark:text-white mb-4">
```
Comment (RajatRajdeep) [inline src/components/MeetOrganizers.jsx]: - There should be only one h1 in the page. Can you make it h2?
- Can you change the text color to primary/secondary for light mode. The current one doesn't match with the theme.
- Can you make the heading center aligned and above the logo and content?
- Use `dark:text-gray-50`. This is being used everywhere.

File: src/components/MeetOrganizers.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { ABOUT_HYDPY } from '@/details';
+import Image from 'next/image';
+
+export default function MeetOrganizers() {
+  return (
+    <div className="flex flex-col lg:flex-row justify-center items-center p-6 pt-12 pb-12 border-t-2 border-gray-300">
```
Comment (RajatRajdeep) [inline src/components/MeetOrganizers.jsx]: The border looks odd, The heading should be enough for separation. Feel free to use top padding if required

File: src/components/MeetOrganizers.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+          {ABOUT_HYDPY.title}
+        </h1>
+        <p className="text-lg md:text-xl text-gray-600 dark:text-gray-400 mb-8">
+          {ABOUT_HYDPY.description}
+        </p>
+      </div>
+      <div className="flex flex-col justify-center items-center gap-3">
+        <Image
+          src={ABOUT_HYDPY.hydPyLogoUrl}
+          alt="Event Design"
```
Comment (RajatRajdeep) [inline src/components/MeetOrganizers.jsx]: The alt tag can be stored in the details. The current one is vague, `HydPy logo` can be used instead.

File: src/components/MeetOrganizers.jsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      <div className="max-w-3xl lg:mr-20">
+        <h1 className="text-4xl font-bold text-[#001ccf] dark:text-white mb-4">
+          {ABOUT_HYDPY.title}
+        </h1>
+        <p className="text-lg md:text-xl text-gray-600 dark:text-gray-400 mb-8">
+          {ABOUT_HYDPY.description}
+        </p>
+      </div>
+      <div className="flex flex-col justify-center items-center gap-3">
+        <Image
```
Comment (RajatRajdeep) [inline src/components/MeetOrganizers.jsx]: Can we show this logo above the content in mobile view?

File: src/components/MeetOrganizers.jsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    <div className="flex flex-col lg:flex-row justify-center items-center p-6 pt-12 pb-12 border-t-2 border-gray-300">
+      <div className="max-w-3xl lg:mr-20">
+        <h1 className="text-4xl font-bold text-[#001ccf] dark:text-white mb-4">
+          {ABOUT_HYDPY.title}
+        </h1>
+        <p className="text-lg md:text-xl text-gray-600 dark:text-gray-400 mb-8">
+          {ABOUT_HYDPY.description}
+        </p>
+      </div>
+      <div className="flex flex-col justify-center items-center gap-3">
```
Comment (RajatRajdeep) [inline src/components/MeetOrganizers.jsx]: The logo is being squeezed for lg screen. I think `flex-col md:flex-row` should solve this.

File: src/components/MeetOrganizers.jsx [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+import Image from 'next/image';
+
+export default function MeetOrganizers() {
+  return (
+    <div className="flex flex-col lg:flex-row justify-center items-center p-6 pt-12 pb-12 border-t-2 border-gray-300">
+      <div className="max-w-3xl lg:mr-20">
+        <h1 className="text-4xl font-bold text-[#001ccf] dark:text-white mb-4">
+          {ABOUT_HYDPY.title}
+        </h1>
+        <p className="text-lg md:text-xl text-gray-600 dark:text-gray-400 mb-8">
```
Comment (RajatRajdeep) [inline src/components/MeetOrganizers.jsx]: The font size looks good. Can you update for announcement text in hero banner for consistency?
Creation of Paragraph component for centralisation will help here.

**Review by RajatRajdeep — CHANGES_REQUESTED** [review]

File: data/details.js (hunk 2 above)
Comment (bkamapantula) [reply data/details.js]: +1

File: src/components/MeetOrganizers.jsx (hunk 1 above)
Comment (vamc-k) [reply src/components/MeetOrganizers.jsx]: Used Link here

File: data/details.js (hunk 2 above)
Comment (vamc-k) [reply data/details.js]: Updated title.

File: src/components/MeetOrganizers.jsx (hunk 3 above)
Comment (vamc-k) [reply src/components/MeetOrganizers.jsx]: Updated styling add requested.

File: src/components/MeetOrganizers.jsx (hunk 5 above)
Comment (vamc-k) [reply src/components/MeetOrganizers.jsx]: Updated alt value using data field hydPyLogoAlt

File: src/components/MeetOrganizers.jsx (hunk 4 above)
Comment (vamc-k) [reply src/components/MeetOrganizers.jsx]: Removed border and used padding.

File: src/components/MeetOrganizers.jsx (hunk 6 above)
Comment (vamc-k) [reply src/components/MeetOrganizers.jsx]: Updated styling as mentioned above

File: src/components/MeetOrganizers.jsx (hunk 7 above)
Comment (vamc-k) [reply src/components/MeetOrganizers.jsx]: Updated alignment.

File: src/components/MeetOrganizers.jsx (hunk 8 above)
Comment (vamc-k) [reply src/components/MeetOrganizers.jsx]: Updated fontsize in hero

**Review by RajatRajdeep — APPROVED** [review]
Thanks @vamc-k, The changes looks good to me.

I have made some minor changes as well. Some of them are following:
- Reduce spacing btw Hero Banner and the HydPy section.
- Learn more about HydPy hyperlink text to Dive Into HydPy. As in mobile view the description is after the hyperlink.

**Pull request merged**